src/gluonts/nursery/autogluon_tabular/predictor.py

```diff
@@ -130,6 +130,10 @@ def _to_forecast(
         start_timestamp: pd.Timestamp,
         item_id=None,
     ) -> Iterator[SampleForecast]:
+        if ag_output.size > self.prediction_length:
+            samples = ag_output.transpose()
+        else:
+            samples = ag_output.reshape((1, self.prediction_length))
```

Comment:
This line is happening in any case just below here, is this correct?

Reply:
Sorry I'm not very clear about this comment. What line are you talking about? 
sample output:
`samples=numpy.array([[135.67691040039062, 171.1056671142578, 183.63880920410156, 197.9020233154297, 162.87466430664062, 147.75, 138.01918029785156, 136.3420867919922, 139.5883331298828, 130.777099609375, 100.32427978515625, 106.0, 108.5605239868164, 93.05574798583984, 103.6478042602539, 119.0, 147.0, 118.28972625732422, 117.5522689819336, 121.49999237060547, 113.92756652832031, 131.35231018066406, 144.86109924316406, 154.62594604492188], [175.91529846191406, 208.87501525878906, 217.50601196289062, 224.64553833007812, 190.80322265625, 175.60281372070312, 165.0, 155.90975952148438, 162.31201171875, 159.0, 126.79875946044922, 126.79875946044922, 130.4644317626953, 117.0, 130.80892944335938, 150.4036407470703, 181.0, 143.8137969970703, 137.3653106689453, 149.0178680419922, 141.72166442871094, 170.82974243164062, 190.0, 210.4620819091797], [227.33328247070312, 243.37208557128906, 255.0, 257.380126953125, 215.36550903320312, 204.04937744140625, 198.0, 181.0, 178.6450653076172, 185.9631805419922, 150.58973693847656, 156.0, 160.5, 151.05474853515625, 160.5927734375, 174.84234619140625, 221.3333282470703, 169.0, 164.63075256347656, 184.5719451904297, 185.25, 207.8796844482422, 231.82571411132812, 249.3555908203125]]`

Reply:
What I mean is: line 137, immediately following 136, is doing the same reshaping, so this seems odd

Reply:
I see. Thanks! I'll change it to `QuantileForecast`